src/test/java/lotto/model/LottoTest.java

```diff
@@ -0,0 +1,58 @@
+package lotto.model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class LottoTest {
+
+    @Test
+    void 로또_번호의_개수가_6개가_넘어가면_예외가_발생한다() {
+        assertThatThrownBy(() -> new Lotto(List.of(1, 2, 3, 4, 5, 6, 7)))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @DisplayName("로또 번호에 중복된 숫자가 있으면 예외가 발생한다.")
+    @Test
+    void 로또_번호에_중복된_숫자가_있으면_예외가_발생한다() {
+        assertThatThrownBy(() -> new Lotto(List.of(1, 2, 3, 4, 5, 5)))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
```

Comment:
어떤 Exception 클래스인지, 어떤 메세지를 출력하는지 포함되면 좀 더 정확한 검증이 될 것 같아요!

```suggestion
@DisplayName("로또 번호에 중복된 숫자가 있으면 예외가 발생한다.")
    @Test
    void 로또_번호에_중복된_숫자가_있으면_예외가_발생한다() {
        assertThatThrownBy(() -> new Lotto(List.of(1, 2, 3, 4, 5, 5)))
                .isInstanceOf(LottoException.class)
                .hasMessage(LOTTO_NUMBERS_DUPLICATE.getMessage());
    }
```

이런식으로요!

Reply:
이 부분도 추후 리팩토링을 고려했는데 캐치를 못했던거 같습니다 좋은 정보 감사합니다.